R/deleteGates.R

```diff
@@ -0,0 +1,39 @@
+#' Delete gate
+#'
+#' Deletes a gate or a tailored gate family
+#' Compound gates (quadrant and split) are made up of "sectors." 
+#' Quadrant gates have four sectors (upper-right, upper-left, lower-left, lower-right) and split gates have two sectors (left and right).
+#' In addition to the top-level GID (like simple gates), these gates have model.gids and names lists that specify the GID and name for each sector, in the order shown above. 
+#' Populations using compound gates must reference these sector GIDs; referencing the top-level GID of a compound gate is meaningless.
```

Comment:
I think you can delete these four lines; only need docs here that are relevant to deleting gates. If you want, you could copy what you think is relevant from the API docs though: https://docs.cellengine.com/api/#delete_4.